docs/source/examplecleanup.rst

```diff
@@ -56,53 +77,62 @@ An End-to-End example walkthrough of a tartufo scan and the process of purging t
       cp -r ${GITHUBREPO} backup_${GITHUBREPO}
       java -jar bfg-1.13.0.jar --replace-text remove.txt ${GITHUBREPO}
 
-
 #. Uh Oh!
 
-   Occasionally the results will be to big to process all at once. If that happens simply split up the results and loop through them.
+   Occasionally the results will be too big to process all at once. If that
+   happens, simply split up the results and loop through them.
 
    .. code-block:: console
 
       # occasionally the results will be to big to process all at once
       split -l 200 remove.txt
       for f in x*; do java -jar bfg-1.13.0.jar --replace-text $f ${GITHUBREPO}; done
 
-
 #. Proceed with cleanup/audit
 
-   Now you have removed the low hanging fruit, it's time to look at the tough stuff
+   Now you have removed the low hanging fruit, it's time to look at the tough
+   stuff
 
    .. code-block:: console
 
-      #run tartufo again to check for icky code straglers in active code
-      leftovers=`tartufo --regex --no-cleanup ${GITHUBREPO}`
+      # run tartufo again to check for icky code stragglers in active code
```

Comment:
This type of idiomatic language might be difficult to parse for non-native English speakers.

Reply:
Good point. Do you have some recommended improved verbiage?

Reply:
Maybe `run tartufo again to check for potential secrets in the latest commit`

Reply:
I'm not sure that was the *exact* intent so I left it a bit more open-ended:
> run tartufo again to check for any remaining potential secrets

@jgowdy That work for ya?

Reply:
👍